core/classes/Misc/NamelessOAuth.php

```diff
@@ -46,19 +46,18 @@ public function getProviders(): array
     }
 
     /**
-     * Determine if OAuth is available if at least one provider is setup.
+     * Get or create an instance of a specific provider.
      *
-     * @return bool If any provider is setup
+     * @param string $provider The provider name
+     * @return null|array An array of registered OAuth provider.
      */
-    public function isAvailable(): bool
+    public function getProvider(string $provider): ?array
```

Comment:
@partydragen can you confirm that this is correct please - this was the one merge conflict I had to resolve for this PR

Reply:
I have no idea however it looks to be working